src/components/attendee/Input/Input.tsx

```diff
@@ -69,6 +102,8 @@ export const Input: React.ForwardRefRenderFunction<
           onKeyDown={handleKeyPress}
         />
 
+        {icon && renderIcon(icon, onIconClick)}
```

Comment:
[NB] Some of the designs I've seen have this icon before the input on the left side, is there an easy way to do this or should we mark it OOS?

Reply:
I'd leave it for the next PR. 🙃